config/README.md

```diff
@@ -1,102 +1,103 @@
-# Multi-region cockroachdb setup
+# Deploying a DSS
+
+## Preface
+
+This doc provides a well-lit path for deploying the DSS and its dependencies (
+namely CockroachDB) on Kubernetes. This is not a requirement, and a DSS node
+can join a cluster as long as it meets the following requirements:
+
+*   Every CockroachDB node must advertise a unique and routable address.
+    *   It is preferred to use domain names with unique prefixes and homogenous
+        suffixes, ie: 0.c.dss.interussplatform.com.
+    *   This allows wildcard usage in the CRDB certificates.
+*   Every DSS node should run a minimum of 3 CockroachDB nodes.
+*   At least 3 CockroacbDB addresses must be shared with all participants.
+    *   If not using the recommended hostname prefix above, every cockroachDB
+        hostname must be shared with every participant.
+*   Every DSS node must supply and share their CockroachDB public certificate
```

Comment:
To whom via what means?  If we don't have a protocol for automated sharing yet and the recipients are a well-defined limited set, we can just say they will be shared out-of-band, but we should make these steps as directly actionable as possible.

Reply:
added a bullet point, planning on adding the functionality in the near future